direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/MFA.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package info.seleniumcucumber.userStepDefinitions;
+
+import env.CucumberRoot;
+import env.DriverUtil;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import pages.*;
+
+public class MFA extends CucumberRoot {
+
+    private static final int RETRY_COUNT = 5;
```

Comment:
Add it to configuration?

Reply:
Int ref: [OKTA-401435](https://oktainc.atlassian.net/browse/OKTA-401435)